presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -515,3 +741,137 @@ The following properties allow tuning the :doc:`/functions/regexp`.
     to hit the limit on matches for subsequent rows as well, you want to use the
     correct algorithm from the beginning so as not to waste time and resources.
     The more rows you are processing, the larger this value should be.
+
+Session properties
```

Comment:
This page is already quite long (and getting longer), so we should probably create a new `session-properties` page for these.